devito/ir/iet/analysis.py

```diff
@@ -2,8 +2,9 @@
 from functools import cmp_to_key
 
 from devito.ir.iet import (Iteration, HaloSpot, SEQUENTIAL, PARALLEL, PARALLEL_IF_ATOMIC,
-                           VECTOR, WRAPPABLE, ROUNDABLE, AFFINE, OVERLAPPABLE, hoistable,
-                           useless, MapNodes, Transformer, retrieve_iteration_tree)
+                           VECTOR, TILABLE, WRAPPABLE, ROUNDABLE, AFFINE, OVERLAPPABLE,
+                           hoistable, useless, MapNodes, Transformer,
```

Comment:
nitpicking: I am not sure about the sequence here.  
Should `hoistable` and `useless` be after `Transformer`?  And  `MapNodes` and `Transformer` closer to `Iteration` and `HaloSpot`?